File: Python/Gate Four/level_five.py
def sorted_prime_numbers(numbers):
   

    new_list = []

    for number in numbers:

        for index in range(2, number):

            if index <= 1 and index % number == 0:
        
                new_list.append(number)      

                
                return new_list
     

def negative_numbers(numbers):


    for index in range(len(numbers)):

            if numbers[index] < 0:

                numbers[index] = 0

    return numbers

#

def zero_numbers(numbers):

#             
#    numbers.sort()
#
#    numbers.reverse()
#
#    return numbers
#
    new_list = []

    for index in range(0, len(numbers)):

        for count in range(0, len(numbers)):

            if numbers[index] > numbers[count]:

                new_list = numbers[index]

                numbers[index] = numbers[count]

                numbers[count] = new_list

    return numbers




def return_duplicate_numbers(numbers):

    new_list = []

    for number in numbers:

        for index in range(0,number):

            if index == number:

                new_list.append(number)

    return new_list

File: Javascript/Gate Four/LevelFive.js
function sorted_prime_numbers(numbers){

    let new_list = []

    for(let count = 0; count < numbers.length; count++){

        for(let index = 2; index < count; index++){

            if(index <= 1 && index % count == 0){
        
}
                new_list.push(count)      
}
}
                
                
    return new_list

}

console.log(sorted_prime_numbers([5,9,3,6,2]));




function negative_numbers(numbers){


    for(let index = 0; index < numbers.length; index++){

            if(numbers[index] < 0){

                numbers[index] = 0
}
}
    return numbers

}

console.log(negative_numbers([5,-9,3,-6,2,-1]))



function zero_numbers(numbers){

    new_list = []

    for(let index = 0; index < numbers.length; index++){

       for(let count = 0; count < numbers.length; count++){

            if(numbers[index] > numbers[count]){

                new_list = numbers[index]

                numbers[index] = numbers[count]

                numbers[count] = new_list
}

}
}
    return numbers


}

console.log(zero_numbers([5,0,3,0,2,0]))







function return_duplicate_numbers(numbers){



    for(let index = 0; index < numbers.length; index++){


            if(numbers[index] > 1){

}
             if(numbers[index] = numbers[index]){

            

}
    return numbers[index]

}




}

console.log(return_duplicate_numbers([45,60,3,0,67,2,45,3,22,0]))
